### ISSUE TITLE
[interface-name-prefix] False position when interface already begins with I

### ISSUE THREAD

**Issue by antonsamper** [typescript-eslint/typescript-eslint#216]
**Repro**

```JSON
{
  "parser": "@typescript-eslint/parser",
  "plugins": [
    "@typescript-eslint"
  ],
  "extends": [
    "plugin:@typescript-eslint/recommended"
  ]
}
```

```TS
interface ITest {
    test: string;
}
```

**Expected Result**

Error should not be shown

**Actual Result**

` error    Interface name must not be prefixed with "I"       @typescript-eslint/interface-name-prefix`

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `^1.2.0` |
| `@typescript-eslint/parser`        | `^1.2.0` |
| `TypeScript`                       | `3.2.2` |
| `ESLint`                           | `^5.13.0` |
| `node`                             | `9.2.0` |
| `npm`                              | `5.5.1` |
